lable.rb

```diff
@@ -0,0 +1,12 @@
+class Label
+  def initialize(title, color, _items)
+    @id = Random.rand(1..1000)
+    @title = title
+    @color = color
+    @items = []
+  end
+
+  def add_item(item)
+    @items.push(item) unless @items.include?(item)
```

Comment:
I think there should be some changes in your code. On line 10, you have to push the item into the items array:
`@items.push(item)`, but than add:
`item.label = self`
to make the association.